invenio_drafts_resources/services/records/service.py

```diff
@@ -547,7 +547,7 @@ def import_files(self, identity, id_, uow=None):
 
         # Read draft
         draft = self.draft_cls.pid.resolve(id_, registered_only=False)
-        self.require_permission(identity, "draft_create_files", record=draft)
+        self.require_permission(identity, "update_draft", record=draft)
```

Comment:
this change confuses me.

we are in the `import_files` method and we don't use a corresponding permission. normally the permissions reflect where they are used, so in `create_draft` the `can_create_draft` permission is used, etc. but here we have `import_files` and first it was `can_create_files` and now it is `update_draft`.

ok, `update_draft` seem not unlogical because in the end it updates the draft, but that is not how i would expect the permissions will be used. 

in my opinion we should add a separate permission `can_import_files` for this method

Reply:
btw also `can_draft_create_files` makes sense, because the files are created, but again there are semantics behind which are not the same